driver-core/src/main/com/mongodb/client/model/Projections.java

```diff
@@ -57,6 +58,21 @@ private Projections() {
         return new SimpleExpression<TExpression>(fieldName, expression);
     }
 
+    /**
+     * Creates a projection of fields computed from the given expressions. Projection with an expression is only supported
+     * using the $project aggregation pipeline stage.
+     *
+     * @param fields     the map of fields to expressions
+     * @param <TExpression> the expression type
+     * @return the projection
+     * @see Aggregates#project(Bson)
+     * @see com.mongodb.client.model.expressions.Expressions
+     * @since 4.?
+     */
+    public static <TExpression> Bson computed(final Map<String, TExpression> fields) {
```

Comment:
Similar to the addFields overload, this makes it easier to project multiple computed fields.  Perhaps this method should be called `of`.